### ISSUE TITLE
Test Terminated with Error Code: 1414549496 - Invalid settings?

### ISSUE THREAD

**Issue by RimasRaV1** [Alkl58/NotEnoughAV1Encodes#162]
When testing Advanced settings on svt-av1 (AV1 FFmpeg) encoder, I get an error "Test Terminated with Error Code: 1414549496 - Invalid settings?" even with default Advanced settings. If I try to encode a video, it does not produce webm video file, see the log attached.
Therefore I cannot encode a video with the Advanced options turned on. If I turn off Advanced settings, Everything encodes noramlly.

**To Reproduce**
1. Load  video file
2. Save Destination of a  file
3. Go to 'Video' tab
4. Select 'Encoder': 'svt-av1 (AV1 FFmpeg)'
5. Click on 'Advanced settings'
6. Go to 'Advanced' tab
7. Click on 'Test settings'
8. See error

**Expected behavior**
Test should be successful/I should be able to encode a file with Advanced settings.

**Screenshots**
![image](https://github.com/Alkl58/NotEnoughAV1Encodes/assets/101190509/7e41c6cd-07a3-4393-a00a-6612f283ec75)

**Log File**
[DSC_0803test.mkv.log](https://github.com/Alkl58/NotEnoughAV1Encodes/files/15024923/DSC_0803test.mkv.log)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro 23H2
 - NEAV1E Version: 2.1.6
 - FFmpeg version 2024.04.18
 - SVT-AV1 version 2024.02.01

**Additional context**
Tested different inputs on 2 Win 11 PC's. It doesn't work.

**Comment by Alkl58** [comment]
Apparently ffmpeg changed the option list for svt-av1, according to the [docs](https://www.ffmpeg.org/ffmpeg-all.html#libsvtav1) the options you have are supported (`tile_rows`/`tile_columns`/`la_depth`).

However when calling ffmpeg directly (`ffmpeg -help encoder=libsvtav1`):
```cmd
libsvtav1 AVOptions:
  -preset            <int>        E..V....... Encoding preset (from -2 to 13) (default -2)
  -crf               <int>        E..V....... Constant Rate Factor value (from 0 to 63) (default 0)
  -qp                <int>        E..V....... Initial Quantizer level value (from 0 to 63) (default 0)
  -svtav1-params     <dictionary> E..V....... Set the SVT-AV1 configuration using a :-separated list of key=value parameters
```

Basically their documentation is now wrong. I will shorty fix this, as those options can be put under `svtav1-params`.

**Comment by NoUserNameForYou** [comment]
While the developer fixes the problem, you may use this in the meantime https://github.com/GyanD/codexffmpeg/releases/tag/6.1.1
Put in apps/ffmpeg folder.

**Comment by Alkl58** [comment]
Fixed in the new [v2.1.7](https://github.com/Alkl58/NotEnoughAV1Encodes/releases/tag/v2.1.7) release.
